USAGE.md

```diff
@@ -45,9 +46,9 @@ See the [Hue Emulation](https://www.openhab.org/addons/integrations/hueemulation
     * The interactive REST API interface available through the openHAB 2 dashboard
     * PaperUI does not yet allow manipulation of tags
 
-#### Item Label Recommendation
+#### Item Label Recommendation and Requirement
```

Comment:
I don't think this belongs here, this is a section header which summarizes the info proceeding it.